package.json

```diff
@@ -3,29 +3,37 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "etherscan": "hardhat etherscan-verify --network",
-    "foundry:compile": "forge build --optimize --optimizer-runs 1000",
-    "foundry:coverage": "forge coverage --report lcov && forge coverage",
-    "foundry:deploy": "forge script --broadcast --verify -vvvv",
-    "foundry:fork": "source .env && anvil --fork-url $ETH_NODE_URI_MAINNET ",
-    "foundry:fork:block": "source .env && anvil --fork-url $ETH_NODE_URI_MAINNET --fork-block-number ",
-    "foundry:gas": "forge test --gas-report",
-    "foundry:run": "docker run -it --rm -v $(pwd):/app -w /app ghcr.io/foundry-rs/foundry sh",
-    "foundry:simulate": "forge script scripts/foundry/Simulate.s.sol --broadcast -vvvv",
-    "foundry:simulate:fork": "forge script scripts/foundry/Simulate.s.sol --fork-url fork --broadcast -vvvv",
-    "foundry:test": "forge test -vvv",
-    "hardhat:compile": "hardhat compile",
-    "hardhat:coverage": "hardhat coverage",
-    "hardhat:deploy": "hardhat deploy --network mainnet --tags",
-    "hardhat:test": "hardhat test",
-    "check-upgradeability": "hardhat run scripts/checkUpgradeability.ts",
-    "license": "hardhat prepend-spdx-license",
-    "node:fork": "FORK=true hardhat node --tags",
-    "node:instant": "FORK=true hardhat node --tags instant",
-    "size": "forge build --sizes",
-    "lint": "yarn lint:sol && yarn lint:js:fix",
-    "lint:js:fix": "eslint --ignore-path .gitignore --fix --max-warnings 30 'test/**/*.{js,ts}' '*.{js,ts}'",
-    "lint:sol": "solhint --fix --max-warnings 20 \"contracts/**/*.sol\""
+    "ci:coverage": "forge coverage --report lcov && yarn lcov:clean",
+    "coverage": "FOUNDRY_PROFILE=dev forge coverage --report lcov && yarn lcov:clean && yarn lcov:generate-html",
+    "compile": "forge build",
+    "compile:dev": "FOUNDRY_PROFILE=dev forge build",
+    "deploy": "forge script --skip test --broadcast --verify --slow -vvvv --rpc-url mainnet scripts/DeployMockAgEUR.s.sol",
+    "deploy:fork": "source .env && forge script --skip test  --slow --fork-url fork --broadcast scripts/DeployMockAgEUR.s.sol -vvvv",
+    "gas": "yarn test --gas-report",
+    "fork": "source .env && anvil --fork-url $ETH_NODE_URI_MAINNET",
+    "fork:arbitrum": "source .env && anvil --fork-url $ETH_NODE_URI_ARBITRUM",
```

Comment:
Open question, but maybe there are clever ways to create scripts to fork on different chains?

Reply:
maybe some bash scripts